docs/developer/decision-records/2022-07-01-get-claims/README.md

```diff
@@ -61,3 +61,44 @@ For each VerifiableCredential:
    10. From the issuer DID Document, participant B extracts the public key of the issuer and uses it to verify the JWS signature.  
    11. If the signature is successfully verified, it extracts the claims of the participant, each claim needs to be associated with its issuer.  
 12. The list of the claims will be used as input to apply the access policies.
+
+## Claims format
+
+The IdentityHubCredentialsVerifier returns a Map<String, Object>, it contains verifiable credentials with the following format:
+
+```json
+<verifiable-credential-id>: {
```

Comment:
nit: this is shown as an error in GitHub web and VSCode. Consider making the JSON valid. e.g.
```
  "sub": "<subject> (part of the JWT claims)"
```